corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap3/modal_report_issue.html

```diff
@@ -25,16 +25,17 @@ <h4 class="modal-title">{% trans "Report an Issue with CommCare HQ" %}</h4>
                 </div>
                 <div class="modal-body">
                     <div class="form-horizontal">
-                        <div class="form-group" id="bug-report-subject-form-group">
+                        <div class="form-group" id="bug-report-subject-form-group" data-bind="css: { 'has-error has-feedback': hasSubjectError }">
```

Comment:
just confirming that these classes are added if `hasSubjectError` is `true`?

Reply:
That is correct. You can also take a look at the relevant [knockoutJS documentation](https://knockoutjs.com/documentation/css-binding.html) which shows an example of using a true/false variable in this manner.